public/app/features/manage-dashboards/components/ImportDashboardForm.tsx

```diff
@@ -136,6 +147,18 @@ export const ImportDashboardForm: FC<Props> = ({
             </Field>
           );
         })}
+      <ImportDashboardLibraryPanelsList
+        inputs={newLibraryPanels}
+        label="New library panels"
+        description="Lists all new library panels. These library panels will be imported."
+        folderName={watchFolder.title}
+      />
+      <ImportDashboardLibraryPanelsList
+        inputs={existingLibraryPanels}
+        label="Existing library panels"
+        description="Lists all library panels that already exist. These library panels won't be re-imported."
```

Comment:
@achatterjee-grafana I could use an extra pair of eyes on these two texts, thank you.